index.js

```diff
@@ -63,9 +64,10 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        file[name] = clone(included[filename]);
+        file[name].contents = included[filename].contents;
```

Comment:
Shouldn't need to assign the `contents` key, since it will come with it from the `clone()` call.